src/main/java/bridge/view/OutputView.java

```diff
@@ -0,0 +1,115 @@
+package bridge.view;
+
+import bridge.domain.dto.MapDto;
+import bridge.domain.dto.OutcomDto;
+import bridge.domain.dto.Result;
+import java.util.ArrayList;
+import java.util.List;
+
+public class OutputView {
+
+    public static void printGameStart() {
+        System.out.println(OutMessage.GAME_START.getPrint());
+        printEmpty();
+    }
+
+    public static void printEmpty() {
+        System.out.println();
+    }
+
+    public static void printRequestBridgeSize() {
+        System.out.println(OutMessage.REQUEST_BRIDGE_SIZE.getPrint());
+    }
+
+    public static void printRequestMoving() {
+        System.out.println(OutMessage.REQUEST_MOVING.getPrint());
+    }
+
+    public static void printRequestGameCommand() {
+        System.out.println(OutMessage.REQUEST_GAME_COMMAND.getPrint());
+    }
+
+    public static void printMap(final MapDto mapDto) {
+        System.out.print(Map.PREFIX);
+        List<String> results = printMapOne(mapDto.getUpResult());
+        System.out.print(String.join(Map.DELIMITER, results));
+        System.out.println(Map.SUFFIX);
```

Comment:
단순히 한 줄을 출력하기 위한 코드라기엔 코드 라인이 너무 긴 것 같아요.
자바 버전이 올라가면서 한 줄에서 여러번 `String` 변수들의 `+` 연산을 수행해 주는 것은 큰 오버헤드가 발생하지 않는다고 합니다.
이와 관련해서는 프리코스 커뮤니티에 좋은 아티클이 존재해 공유 드립니다!
[String보다 StringBuilder를 써야한다? String.join은 어떨까?](https://discord.com/channels/1149138870433230900/1170269917963096154)
```suggestion
        List<String> results = printMapOne(mapDto.getUpResult());
        System.out.println(Map.PREFIX + String.join(Map.DELIMITER, results) + Map.SUFFIX);
```